operations/reservations.md

```diff
@@ -124,7 +126,6 @@ Returns all reservations within scope of the Access Token, filtered according to
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
 | `Id` | string | required | Unique identifier of the reservation. |
-| `EnterpriseId` | string | required | Unique identifier of the [Enterprise](enterprises.md#enterprise). |
```

Comment:
Should we mention removing of this in the docs as well? It was never present there

Reply:
Do you mean mentioning it in the changelog? I think we can mention it. But did we decide not to fix the code to return this field?